test/extended/jenkins/plugin.go

```diff
@@ -125,6 +138,11 @@ var _ = g.Describe("[jenkins] openshift pipeline plugin", func() {
 		err = waitForJenkinsActivity(fmt.Sprintf("http://%s", hostPort), "", 200)
 		o.Expect(err).NotTo(o.HaveOccurred())
 
+		if testingSnapshot {
+			// for the test image, confirm that a snapshot version of the plugin is running in the jenkins image we'll test against
+			err = waitForJenkinsActivity(fmt.Sprintf("http://%s/pluginManager/api/xml?depth=1", hostPort), "SNAPSHOT", 200)
```

Comment:
Perhaps ... I think I can confirm that openshift-pipeline-plugin is the SNAPSHOT.